pkg/k8s/watchers/node.go

```diff
@@ -41,9 +34,9 @@ func (k *K8sWatcher) RegisterNodeSubscriber(s subscriber.Node) {
 
 // The KVStoreNodeUpdater interface is used to provide an abstraction for the
 // nodediscovery.NodeDiscovery object logic used to update a node entry in the
-// KV store.
+// KVStore.
 type KVStoreNodeUpdater interface {
```

Comment:
nit: should we rename this given that it's used both for updating the k8s `CiliumNode` and the KVStore entry?